.changes/fix-dynamic-linking.md

```diff
@@ -0,0 +1,12 @@
+
+---
+"libappindicator": major
```

Comment:
the bump should be `minor` instead, we're not changing this crate to stable yet.